packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js

```diff
@@ -406,36 +413,49 @@ export class SearchFormImpl extends React.PureComponent {
               </div>
             }
           >
-            <Field
-              name="endDate"
-              type="date"
-              component={AdaptedInput}
-              placeholder="End Date"
-              props={{ disabled }}
-            />
-            <Field name="endDateTime" type="time" component={AdaptedInput} props={{ disabled }} />
+            <Row gutter={16}>
+              <Col className="gutter-row" span={12}>
+                <Field
+                  name="endDate"
+                  type="date"
+                  component={AdaptedInput}
+                  placeholder="End Date"
+                  props={{ disabled }}
+                />
+              </Col>
+
+              <Col className="gutter-row" span={12}>
+                <Field name="endDateTime" type="time" component={AdaptedInput} props={{ disabled }} />
+              </Col>
+            </Row>
           </FormItem>,
         ]}
 
-        <FormItem label="Min Duration">
-          <Field
-            name="minDuration"
-            component={ValidatedAdaptedInput}
-            placeholder={placeholderDurationFields}
-            props={{ disabled }}
-            validate={validateDurationFields}
-          />
-        </FormItem>
-
-        <FormItem label="Max Duration">
-          <Field
-            name="maxDuration"
-            component={ValidatedAdaptedInput}
-            placeholder={placeholderDurationFields}
-            props={{ disabled }}
-            validate={validateDurationFields}
-          />
-        </FormItem>
+        <Row gutter={16}>
+          <Col className="gutter-row" span={12}>
+            <FormItem label="Max Duration">
+              <Field
+                name="maxDuration"
+                component={ValidatedAdaptedInput}
+                placeholder={placeholderDurationFields}
+                props={{ disabled }}
+                validate={validateDurationFields}
+              />
+            </FormItem>
+          </Col>
+
+          <Col className="gutter-row" span={12}>
+            <FormItem label="Min Duration">
+              <Field
+                name="minDuration"
+                component={ValidatedAdaptedInput}
+                placeholder={placeholderDurationFields}
+                props={{ disabled }}
+                validate={validateDurationFields}
+              />
+            </FormItem>
+          </Col>
+        </Row>
```

Comment:
I think there was also an ask to make the Search button into different color.

Reply:
done 😄